app/scripts/modules/netflix/chaosMonkey/chaosMonkeyExceptions.directive.js

```diff
@@ -36,7 +36,9 @@ module.exports = angular
           this.accounts = details;
           this.regionsByAccount = {};
           details.forEach((account) => {
-            this.regionsByAccount[account.name] = ['*'].concat(account.regions.map((region) => region.name));
+            if (account.regions) {
+              this.regionsByAccount[account.name] = ['*'].concat(account.regions.map((region) => region.name));
+            }
```

Comment:
That will stop the exception from being thrown, but you'll still have those accounts listed in the dropdown, I think. If you changed #L36 to `this.accounts = details.filter(account => account.regions);`, you could skip the changes to L39 and only show AWS accounts to the user.